src/features/zoom_to/readme.md

```
@@ -0,0 +1,15 @@
+## Zoom to selected area
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)